code/modules/modular_computers/file_system/programs/command/away_manifest.dm

```diff
@@ -0,0 +1,109 @@
+/datum/computer_file/program/away_manifest
+	filename = "awayshuttlemanifest"
+	filedesc = "Shuttle Manifest Program"
+	extended_desc = "Used to view the manifest of the Horizon's shuttles."
+	program_icon_state = "menu"
+	program_key_icon_state = "lightblue_key"
+	color = LIGHT_COLOR_BLUE
+	size = 8
+	requires_ntnet = TRUE
+	available_on_ntnet = TRUE
+	usage_flags = PROGRAM_ALL_REGULAR
+	tgui_id = "AwayShuttleManifest"
+	var/datum/record/shuttle_manifest/active_record
+
+/datum/computer_file/program/away_manifest/ui_data(mob/user)
+	var/list/data = list()
+	
+	if(active_record)
+		data["active_record"] = list(
+			"name" = active_record.name,
+			"shuttle" = active_record.shuttle,
+			"id" = active_record.id
+		)
+	else
+		var/list/allshuttles = list()
+		for (var/datum/record/shuttle_manifest/m in SSrecords.shuttle_manifests)
+			allshuttles += list(list(
+				"name" = m.name,
+				"shuttle" = m.shuttle,
+				"id" = m.id
+			))
+		data["shuttle_manifest"] = allshuttles
+		data["active_record"] = null
+	return data
+	
+/datum/computer_file/program/away_manifest/ui_act(action, list/params, datum/tgui/ui, datum/ui_state/state)
+	. = ..()
+	if(.)
+		return
+		
+	switch(action)
+		if("am_menu")
+			active_record = null
+			SStgui.update_uis(computer)
```

Comment:
All you need to do is `return TRUE` here

Reply:
Alternatively, if you don't need to return (like I think you do in some places), do `. = TRUE`